aviary/examples/external_subsystems/engine_NPSS/NPSS_Model/Maps_and_Elements/npss.view_Aviary

```diff
@@ -0,0 +1,23 @@
+// <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+//             Aviary Format DataViewer Specification
+// <<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<<
+
+OutFileStream AviarysheetStream { filename = "./NPSS_Model/Output/RefEngine.outputAviary"; } 
```

Comment:
this file path might need to be updated?

Reply:
Will the file paths within NPSS make the tests fail?    I need to look into how to make these explicit.

Reply:
explicit update checked in.